monkey/infection_monkey/example.conf

```diff
@@ -103,9 +100,9 @@
     "use_file_logging": true,
     "victims_max_exploit": 7,
     "victims_max_find": 30,
-    "post_breach_actions" : [],
-    "custom_PBA_linux_cmd" : "",
-    "custom_PBA_windows_cmd" : "",
-    "PBA_linux_filename" : "None",
-    "PBA_windows_filename" : "None"
-}
+    "post_breach_actions" : []
+    custom_PBA_linux_cmd = ""
```

Comment:
Pretty sure that should be with "" and with :, for it to be a valid json

Reply:
Yes! was asked to move this to another PR